src/main/java/internet/market/model/Order.java

```diff
@@ -0,0 +1,38 @@
+package internet.market.model;
+
+import java.util.List;
+
+public class Order {
+    Long id;
+    List<Product> products;
+    Long userId;
+
+    public Order(List<Product> products, Long userId) {
+        this.products = products;
```

Comment:
here you probably need only user id to get the specific order

Reply:
done